agcaballero/portfolio/src/main/webapp/script.js

```diff
@@ -72,6 +72,129 @@ function testRandomPost(trials = 10000) {
 
 testRandomPost(10000);  // runs 10000 trials on randomPost()
 
+/** Creates a Map using Map API */
```

Comment:
Some more detail may be helpful, for example, letting the reader know that the map contains preset markers showing photos taken at different locations.

Reply:
I added more details. Thank you!

Reply:
Done